facebook/src/com/facebook/share/model/GameRequestContent.java

```diff
@@ -21,37 +21,101 @@
 package com.facebook.share.model;
 
 import android.os.Parcel;
+import android.os.Parcelable;
 
-import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
 
 /**
  * Describes the content that will be displayed by the GameRequestDialog
  */
 public final class GameRequestContent implements ShareModel {
-    public enum ActionType {
-        SEND,
-        ASKFOR,
-        TURN,
+    public enum ActionType implements Parcelable {
```

Comment:
Instead of making this parcelable, which adds a lot of boilerplate code, what about doing what we do in [AppGroupCreationContent](
https://github.com/facebook/facebook-android-sdk/blob/001984c85bbb5ccb7a3df4e33fe3090d22a8ac88/facebook/src/com/facebook/share/model/AppGroupCreationContent.java#L42)

And just add them as serializable?